src/main/drivers/timer.c

```diff
@@ -567,10 +569,11 @@ volatile timCCR_t* timerChCCRLo(const timerHardware_t *timHw)
 {
     return (volatile timCCR_t*)((volatile char*)&timHw->tim->CCR1 + (timHw->channel & ~TIM_Channel_2));
 }
+#endif
 
 volatile timCCR_t* timerChCCR(const timerHardware_t *timHw)
 {
-    return (volatile timCCR_t*)((volatile char*)&timHw->tim->CCR1 + timHw->channel);
+    return (volatile timCCR_t*)(&timHw->tim->CCR1 + timHw->channel);
```

Comment:
My bad. Since timerHardware_s.channel is 1-origin, this must be
```
return (volatile timCCR_t*)(&timHw->tim->CCR1 + (timHw->channel - 1));
```

Reply:
Originally `channel` was taken from ST macros ...
One based channel indexing is a bit strange (but matches channel names)

Reply:
@blckmn Same fix required for `timer_hal.c`, too.

Reply:
done :)